Dockerfile

```diff
@@ -86,7 +86,7 @@ ENV POSTGRES_DB=mobsf
 ENV POSTGRES_HOST=postgres
 
 # Check if Postgres support needs to be enabled
-RUN ./scripts/postgres_support.sh $POSTGRES
+RUN cd scripts && ./postgres_support.sh $POSTGRES
```

Comment:
Thanks for identifying this. A recent Dockerfile QA caused this. I think It's cleaner to fix it here:
https://github.com/MobSF/Mobile-Security-Framework-MobSF/blob/3aa18db62775aee85e9bf8e82fc4fa2f85378531/scripts/postgres_support.sh#L8-L10

Removing the `../` would do.